packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -154,6 +154,10 @@ contract L2Genesis is Script {
             return;
         }
 
+        if (forkEquals(_fork, Fork.INTEROP)) {
+            return;
+        }
+
```

Comment:
There's no `activateInterop();` that we have to call before, like for Isthmus and Fjord?

Reply:
No. Those are calling a function on the L1 block attributes contract and there isn't a need for that in this fork because nothing changed.  Holocene was the same and doesn't have an activateHolocene call.